rpc/api.proto

```diff
@@ -255,6 +255,8 @@ message GetTransactionRequest {
 }
 message GetTransactionResponse {
 	TransactionDetails transaction = 1;
+	int32 confirmations = 2;
```

Comment:
Just a nit.

Do you specifically need `confirmations`? Otherwise, maybe it would be better to add `height` and `timestamp` to keep it in line with the `BlockDetails` response from `GetTransactions`.

Reply:
This was an addition for @mbelop.

The number of confirmations is being used to determine how safe it is to assume this transaction won't be reorged out forever.  Without knowledge of the current main chain height, and without knowledge that this transaction is actually mined in the main chain and not a sidechain, the height is not enough.

Reply:
Additionally the transaction could be mined in many different chain forks at different heights.  The number of confirmations is relative to the current best chain, which is a less ambiguous result.

Reply:
Makes sense, thanks for the clarification!